diff --git a/CMakeLists.txt b/CMakeLists.txt
index 80f87ae..761e0d3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -242,24 +242,24 @@ if(ENABLE_PYTHON)
 endif(ENABLE_PYTHON)
 
 # install library
-install(TARGETS netconf2 DESTINATION ${CMAKE_INSTALL_LIBDIR})
+install(TARGETS netconf2 DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT libnetconf2)
 
 # install headers
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/nc_client.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/nc_server.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
-install(FILES ${headers} DESTINATION ${INCLUDE_INSTALL_SUBDIR})
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/nc_client.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR} COMPONENT libnetconf2)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/nc_server.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR} COMPONENT libnetconf2)
+install(FILES ${headers} DESTINATION ${INCLUDE_INSTALL_SUBDIR} COMPONENT libnetconf2)
 
 # install schemas
 install(
     CODE "file(GLOB yin_schemas \"${CMAKE_SOURCE_DIR}/schemas/*.yin\")"
-    CODE "file(INSTALL \${yin_schemas} DESTINATION ${CMAKE_INSTALL_PREFIX}/${DATA_INSTALL_DIR})"
+    CODE "file(INSTALL \${yin_schemas} DESTINATION ${CMAKE_INSTALL_PREFIX}/${DATA_INSTALL_DIR})" COMPONENT libnetconf2
 )
 
 # install pkg-config file
 find_package(PkgConfig)
 if(PKG_CONFIG_FOUND)
     configure_file("libnetconf2.pc.in" "libnetconf2.pc" @ONLY)
-    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/libnetconf2.pc" DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig")
+    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/libnetconf2.pc" DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig" COMPONENT libnetconf2)
     # check that pkg-config includes the used path
     execute_process(COMMAND ${PKG_CONFIG_EXECUTABLE} --variable pc_path pkg-config RESULT_VARIABLE RETURN OUTPUT_VARIABLE PC_PATH ERROR_QUIET)
     if(RETURN EQUAL 0)
@@ -296,3 +296,7 @@ add_custom_target(cleancache
 
 # uninstall
 add_custom_target(uninstall "${CMAKE_COMMAND}" -P "${CMAKE_MODULE_PATH}/uninstall.cmake")
+
+include(PackagingLibNetconf)
+include(Packaging)
+make_packages()
\ No newline at end of file
