diff --git a/cli/CMakeLists.txt b/cli/CMakeLists.txt
index 1e99724..7a81513 100644
--- a/cli/CMakeLists.txt
+++ b/cli/CMakeLists.txt
@@ -69,7 +69,8 @@ if(HAVE_MKSTEMPS)
 endif(HAVE_MKSTEMPS)
 
 # install binary
-install(TARGETS netopeer2-cli DESTINATION ${CMAKE_INSTALL_BINDIR})
+install(TARGETS netopeer2-cli DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT
+  netopeer2-cli)
 
 # man -> html
 find_program(ROFF2HTML roff2html)
@@ -81,7 +82,8 @@ else()
 endif()
 
 # install doc (man)
-install(FILES ${PROJECT_SOURCE_DIR}/doc/${PROJECT_NAME}.1 DESTINATION ${CMAKE_INSTALL_MANDIR}/man1)
+install(FILES ${PROJECT_SOURCE_DIR}/doc/${PROJECT_NAME}.1 DESTINATION
+  ${CMAKE_INSTALL_MANDIR}/man1 COMPONENT netopeer2-cli)
 
 # clean cmake cache
 add_custom_target(cleancache
@@ -92,3 +94,7 @@ add_custom_target(cleancache
 
 # uninstall
 add_custom_target(uninstall "${CMAKE_COMMAND}" -P "${CMAKE_MODULE_PATH}/uninstall.cmake")
+
+include(PackagingNetopeer)
+include(Packaging)
+make_packages()
diff --git a/server/CMakeLists.txt b/server/CMakeLists.txt
index f7c6567..9bca82d 100755
--- a/server/CMakeLists.txt
+++ b/server/CMakeLists.txt
@@ -153,7 +153,8 @@ if(GENERATE_HOSTKEY)
 endif()
 
 # install the binary, required modules, and default configuration
-install(TARGETS netopeer2-server DESTINATION ${CMAKE_INSTALL_BINDIR})
+install(TARGETS netopeer2-server DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT
+  netopeer2-server)
 if(INSTALL_MODULES)
     install(CODE "
         message(STATUS \"Installing sysrepo modules...\")
@@ -172,13 +173,15 @@ endif()
 if(GENERATE_HOSTKEY)
     install(CODE "
         message(STATUS \"Generating a new RSA host key \\\"genkey\\\"...\")
-        execute_process(COMMAND ${PROJECT_SOURCE_DIR}/merge_hostkey.sh ${SYSREPOCFG} ${OPENSSL})
+        execute_process(COMMAND ${PROJECT_SOURCE_DIR}/merge_hostkey.sh
+          ${SYSREPOCFG} ${OPENSSL} COMPONENT netopeer2-server)
     ")
 endif()
 if(MERGE_LISTEN_CONFIG)
     install(CODE "
         message(STATUS \"Merging default server listen configuration...\")
-        execute_process(COMMAND ${PROJECT_SOURCE_DIR}/merge_config.sh ${SYSREPOCFG} ${DEFAULT_HOSTKEY})
+        execute_process(COMMAND ${PROJECT_SOURCE_DIR}/merge_config.sh
+          ${SYSREPOCFG} ${DEFAULT_HOSTKEY} COMPONENT netopeer2-server)
     ")
 endif()
 
@@ -195,3 +198,7 @@ add_custom_target(cleancache
 
 # uninstall
 add_custom_target(uninstall "${CMAKE_COMMAND}" -P "${CMAKE_MODULE_PATH}/uninstall.cmake")
+
+include(PackagingNetopeer)
+include(Packaging)
+make_packages()
